ocw_import/api.py

```diff
@@ -40,6 +40,19 @@
 ]
 
 
+def parse_date(date_string):
+    """
+    Small utility function for calling dateutil.parser.parse, done so we can
+    mock out date parsing in unit tests
+
+    Args:
+        date_string (str): A date string to parse
+    Yeilds:
```

Comment:
It spelled `Yields` but here it should be:

    Returns:
        datetime: A date object

Or alternatively, is there a type annotation we can use for datetime?